lib/iris/cube.py

```diff
@@ -1627,9 +1627,17 @@ def dtype(self):
     @property
     def fill_value(self):
         """
-        The fill value for the masked data array of this
-        :class:`~iris.cube.Cube`. If ``None``, the default NumPy fill value for
-        the dtype of the masked data array will be used.
+        A value suitable for filling in missing (i.e. masked) datapoints of
+        this :class:`~iris.cube.Cube` s data.
```

Comment:
Could you please try and avoid forms like `:class:~iris.cube.Cube s` as it just looks a little bad...